sui/its.js

```diff
@@ -5,13 +5,18 @@ const { addBaseOptions, addOptionsToCommands, getWallet, printWalletInfo, broadc
 const { bcs } = require('@mysten/sui/bcs');
 
 const SPECIAL_CHAINS_TAGS = {
-    ALL_EVM: 'all-evm', // All EVM chains that have InterchainTokenService deployed
+    ALL: 'all',
```

Comment:
This change should be in a separate PR than the above refactor since they're unrelated and can be addressed independently without blocking each other